raiden/tasks.py

```diff
@@ -95,6 +98,27 @@ def check_gas_reserve(raiden):
         gevent.sleep(CHECK_GAS_RESERVE_INTERVAL)
 
 
+def check_rdn_deposits(raiden, user_deposit_proxy: UserDeposit):
+    """ Check periodically for RDN deposits in the user-deposits contract """
+    while True:
+        rei_balance = user_deposit_proxy.effective_balance(raiden.address, "latest")
```

Comment:
Please, don't give an object like `raiden` to a function just to get its address, we should definitely apply law of demeter on our code.